webui/react/src/hooks/useFeature.ts

```diff
@@ -1,25 +1,32 @@
 import queryString from 'query-string';
+import { useEffect, useState } from 'react';
 import { useLocation } from 'react-router-dom';
 
 import { useStore } from 'contexts/Store';
 
 interface FeatureHook {
-  isOnRbac: () => boolean
-}
-
-interface Queries {
-  rbac?: string;
+  isOn: (ValidFeature: string) => boolean
 }
 
 const useFeature = (): FeatureHook => {
-  return { isOnRbac: () => IsOnRbac() };
+  const location = useLocation();
+  const [ features, setFeatures ] = useState<string[]>([]);
```

Comment:
You don't need any state for this hook.

Reply:
All you actually have to do is check ``queryParams[`f_${feature}`] === 'on'`` and then have special handling when `feature === 'rbac'` to check the master config info.

Reply:
Yes, this is better, thanks!